teachertool/src/global.scss

```diff
@@ -0,0 +1,334 @@
+/*******************************/
+/*****  GLOBAL VARIABLES  ******/
+/*******************************/
+
+:root {
+    --frame-loader-zindex: 30;
+    --above-frame-zindex: 50;
+    --modal-dimmer-zindex: 100;
+    --fullscreen-modal-zindex: 90;
+}
+
+/*******************************/
+/*********  RESETS  ************/
+/*******************************/
+
+* {
+    // undo semantic.css defaults
+    min-height: initial;
+    min-width: initial;
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+* {
+    margin: 0;
+}
+
+img,
+picture,
+video,
+canvas,
+svg {
+    display: block;
+    max-width: 100%;
+}
+
+input,
+button,
+textarea,
+select {
+    font: inherit;
+}
+
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    overflow-wrap: break-word;
+}
+
+body {
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    font-family: var(--pxt-page-font);
+    background: var(--pxt-page-background);
+    color: var(--pxt-page-foreground);
+    line-height: 1.5;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    // undo semantic.css defaults
+    display: initial;
+    font-size: initial;
+}
+
+code {
+    font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
+}
+
+/*******************************/
+/*********  GLOBALS  ***********/
+/*******************************/
+
+#root {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 100%;
+    overflow: hidden;
+    isolation: isolate;
+}
+
+.noclick {
+    pointer-events: none;
+    cursor: default;
+}
+
+/**********************************************/
+/********* THINGS THAT NEED HOMES *************/
+/**********************************************/
+
+.btn-primary {
+    transition: all 0.1s ease;
+    background-color: var(--pxt-button-primary-background);
+    color: var(--pxt-button-primary-foreground);
+    border-width: 1px;
+    border-style: solid;
+    border-color: var(--pxt-button-primary-accent);
+}
+
+.btn-secondary {
+    transition: all 0.1s ease;
+    background-color: var(--pxt-button-secondary-background);
+    color: var(--pxt-button-secondary-foreground);
+    border-width: 1px;
+    border-style: solid;
+    border-color: var(--pxt-button-secondary-accent);
+}
+
+.btn-inline {
+    transition: all 0.1s ease;
+    background-color: var(--pxt-content-background);
+    color: var(--pxt-content-foreground);
+    border-width: 1px;
+    border-style: solid;
+    border-color: var(--pxt-content-foreground);
+    &:hover {
+        filter: none;
+        background-color: var(--pxt-headerbar-background-glass);
+    }
+}
+
+/*******************************/
+/******* CATALOG STYLING  ******/
```

Comment:
Should the styling sections from here down (other than mobile, small desktop view) be separate modules or is there a reason that we left them in global styles?